code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -245,4 +255,43 @@
 	for(var/i in vines)
 		qdel(i)
 	return ..()
+
+/mob/living/simple_animal/hostile/venus_human_trap/UnarmedAttack(atom/attack_target, proximity_flag, list/modifiers)
+	if(!istype(attack_target, /obj/structure/spacevine))
+		return ..()
+	if(!proximity_flag)
+		return
+	var/obj/structure/spacevine/attacked_spacevine = attack_target
+	if(help_grow <= world.time)
+		help_grow = world.time + 1 SECONDS
+		if(attacked_spacevine.energy >= 2)
+			attacked_spacevine.spread()
```

Comment:
Use the COOLDOWN defines to handles your cooldown, instead of doing it manually.